pytket_qir/qir.py

```diff
@@ -228,6 +245,33 @@ def block_to_circuit(self, block: QirBlock, circuit: Circuit) -> Circuit:
                 circuit.add_gate(op, unitids)
             elif instr.instr.is_qir_call:  # Setting the conditional bit for branching.
                 c_reg_index = instr.instr.call_func_params[0].constant.result_static_id
+            elif instr.instr.is_call:  # WASM external call.
+                if self.wasm_handler is None:
+                    raise ValueError("A WASM file handler must be provided.")
+                func_name = instr.func_name
+                if func_name is None:
+                    raise ValueError("The WASM function call is not defined.")
+                matched_str = re.search("__quantum__(.+?)__(.+?)__(.+)", func_name)
+                if matched_str is None:
+                    raise ValueError(
+                        "The WASM function call name is not propely defined."
+                    )
+                # WASM function call parameters.
+                param_regs = []
+                for c_reg_index in range(len(instr.func_args)):
+                    c_reg_name = "c_reg_wasm" + str(c_reg_index)
+                    param_regs.append(
+                        circuit.add_c_register(c_reg_name, self.wasm_int_type.size)
```

Comment:
There are a couple of other places in this file where 64 seems to be hard-coded: lines 226, 227 and 295.

Reply:
Yes but those are different. There are to hold constants as llvm int64. Maybe worth using another data type to make it more explicit.

Reply:
Done.